os/src/ZipOps.scala

```diff
@@ -257,7 +325,53 @@ object unzip {
       excludePatterns: Seq[Regex] = List(),
       includePatterns: Seq[Regex] = List()
   ): os.Path = {
-    stream(os.read.stream(source), dest, excludePatterns, includePatterns)
+    checker.value.onWrite(dest)
+
```

Comment:
Could you remind me of the reason why we are unable to support permissions and symlinks in `unzip.stream`? I see that the upstream library has a version of `ZipOutputStream` in addition to `ZipFile`. Could we use that to get support for permissions and symlinks?

Reply:
I've tried it and it doesn't work. It could be due to information on permissions and symlinks are stored as external attributes which are stored inside the zip central directory at the end of the zip file, not together with each zip entry. The Apache Commons Compress doc mentions [this](https://commons.apache.org/proper/commons-compress/zip.html) (in the ZipArchiveInputStream vs ZipFile section).

Reply:
> `ZipFile` is able to read the central directory first and provide correct and complete information on any ZIP archive.

Reply:
Got it, that's fine but we just ned to document this limitation somewhere. Maybe on the scaladoc for the `unzip.stream` class?